### DISCUSSION TITLE
Chinese problem，cannot wrap line in correct position, some parameters are invalid

### DISCUSSION THREAD

**Discussion by yg2342423523** [Purfview/whisper-standalone-win#310] [Q&A] [answered]
sir，I have some Chinese recognition needs now, but after trying many parameters, I still can't break the line at the comma. In Chinese, a more reasonable way is to break the line at the comma, but setting the maximum number of characters has no effect. Is it because I'm not doing the right thing. Even the Asian model doesn't seem to be applicable, I really wishyou can help me，thank you！
/[00:00.000 --> 00:02.940] 干得酷2.8, 紧接着2.7就来了。
[00:03.480 --> 00:07.240] 此次更新也是十分重吧, 在大家看不见的地方。
/the correct is this
干得酷2.8, 
紧接着2.7就来了。
此次更新也是十分重吧,
在大家看不见的地方

**Comment by yg2342423523** [comment]
[1.zip](https://github.com/user-attachments/files/17367216/1.zip)

**Reply by Purfview** [reply, marked as answer]
You didn't wrote what settings you have used. Anyway here is a command to make it look like you want:

[1.json.zip](https://github.com/user-attachments/files/17895444/1.json.zip)
[r194.2_to_r194.2.1_patch](https://fromsmash.com/HDB7pHi_qG-bt)

`faster-whisper-xxl.exe 1.json  -f txt --sentence --max_line_width 12 --max_comma_cent 50`

```
[00:00.000 --> 00:01.400]  干得酷2.8,
[00:01.520 --> 00:02.920]  紧接着2.7就来了。
[00:03.480 --> 00:05.840]  此次更新也是十分重磅,
[00:05.960 --> 00:07.240]  在大家看不见的地方。
```
